nmrfx-processor/src/main/java/org/nmrfx/processor/operations/NESTANMR.java

```diff
@@ -131,19 +179,58 @@ public Operation eval(Vec vector) throws ProcessingException {
                 vector.set(i, real, imag);
             }
         } catch (Exception e) {
+            log.error("Error in NESTANMR extend", e);
             throw new ProcessingException(e.getLocalizedMessage());
         }
-        //PyObject obj = interpreter.get("a");
         return this;
     }
 
     @Override
     public Operation evalMatrix(MatrixType matrix) {
+        if (extendMode) {
+            return evalExtendMatrix(matrix);
+        } else {
+            return evalNUSMatrix(matrix);
+        }
+
+    }
+
+    public static int getZfSize(double vecSize, int factor) {
+        return (int) (Math.pow(2, Math.ceil((Math.log(vecSize) / Math.log(2)) + factor)));
+    }
```

Comment:
exact duplicate of static method of MatrixND.getZFSize, only difference is this one is public and the other one is default access...do we need both methods?